sign/anon/enc_test.go

```diff
@@ -80,16 +87,24 @@ func ExampleEncrypt_anonSet() {
 	}
 	fmt.Printf("Decrypted: '%s'\n", string(MM))
 
+	// Output:
 	// Encryption of 'Hello World!':
-	// 00000000  3c 2e 26 55 5e 9c 59 55  68 91 5c 68 19 e3 10 6a  |<.&U^.YUh.\h...j|
-	// 00000010  be 1d 8c fc 52 b1 85 98  31 a9 81 08 24 bb f0 d0  |....R...1...$...|
-	// 00000020  db 47 f9 b3 ee 3b 14 f2  2d 8f 0a c9 83 9d 47 1a  |.G...;..-.....G.|
-	// 00000030  69 0f a4 b2 5b 44 c8 a0  ca 33 1e c6 04 9d 98 35  |i...[D...3.....5|
-	// 00000040  31 cd 3a a9 0b 44 64 d5  a5 54 d4 5d 33 67 9e 2e  |1.:..Dd..T.]3g..|
-	// 00000050  35 e8 05 f3 17 c9 a5 14  9f 5b b6 9a c3 ee 57 54  |5........[....WT|
-	// 00000060  64 2f c2 06 36 ae aa af  f8 61 9d c3 cd 09 c2 d7  |d/..6....a......|
-	// 00000070  74 8d 32 bf 08 cb ef 1d  06 af 35 52 99 1f b1 16  |t.2.......5R....|
-	// 00000080  a7 3c 1b 02 8a 5f bd eb  f0 28 94 df 36 44 07 be  |.<..._...(..6D..|
-	// 00000090  22 01 7c dc ad 06 09 7a  62 8e 45 98              |".|....zb.E.|
+	// 00000000  1d 94 9b 66 98 f7 66 ec  2c 66 7f 00 1a aa 90 59  |...f..f.,f.....Y|
```

Comment:
Do you know why this is nearly double the length?

Reply:
This was because I used xof.KeySize() as the size of the output hash. I have added a new xof.Size() method, which is set correctly for both blake and keccak and is used by sign/anon now.

Reply:
Thumbs-up is: OK, won't bother you with this anymore ;)